docs/lib/auth/fragments/existing-resources.md

```diff
@@ -1,4 +1,14 @@
-Existing Amazon Cognito Identity Pool and User Pool can be used with the Amplify Libraries by referencing them in your `amplifyconfiguration.json` file.
+
+If you are using the Amplify CLI, an existing Amazon Cognito Identity Pool and User Pool can be used with the Amplify Libraries by running:
+
+```bash
+amplify import auth
+```
+
+Refer to [the related CLI documentation](~/cli/auth/import.md) for more details.
```

Comment:
Use AWS writing style guide approach here. Example:
"For more information about <topic>, see [Page title](link)."

Reply:
```suggestion
For more details, see [the related CLI documentation](~/cli/auth/import.md).
```